CONTRIBUTING.md

```diff
@@ -70,7 +70,7 @@ follow the conventions of [Keep A CHANGELOG][kacl].
 
 ## Testing
 
-nix has a test suite that you can run with `cargo test`. Ideally, we'd like pull
+nix has a test suite that you can run with `cargo t --all-features`. Ideally, we'd like pull
```

Comment:
Maybe a full option `cargo test --all-features` is more suitable for a document? I am sorry that I forgot to mention this in #2148